src/lib/core/theming/_palette.scss

```diff
@@ -6,12 +6,16 @@
 // While the contrast colors in the spec are not prescriptive, we use them for convenience.
 
 
-$black-87-opacity: rgba(black, 0.87);
-$white-87-opacity: rgba(white, 0.87);
-$black-12-opacity: rgba(black, 0.12);
-$white-12-opacity: rgba(white, 0.12);
-$black-6-opacity: rgba(black, 0.06);
-$white-6-opacity: rgba(white, 0.06);
+$dark-primary-text: rgba(black, 0.87);
+$dark-secondary-text: rgba(black, 0.54);
+$dark-disabled-text: rgba(black, 0.38);
+$dark-dividers: rgba(black, 0.12);
+$dark-focused: rgba(black, 0.12);
+$light-primary-text: rgba(white, 1);
```

Comment:
`rgba(white, 1)` can just be `white`

Reply:
I felt like it was more clear/explicit that the opacity should be 100% if I typed it out, but I can change it if you prefer it.

Reply:
Yeah, I prefer to use the plain value when it is desired

Reply:
Fixed and rebased